src/libraries/System.Security.Cryptography/ref/System.Security.Cryptography.cs

```diff
@@ -1626,7 +1626,10 @@ public partial class PasswordDeriveBytes : System.Security.Cryptography.DeriveBy
         [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")]
         public byte[] CryptDeriveKey(string? algname, string? alghashname, int keySize, byte[] rgbIV) { throw null; }
         protected override void Dispose(bool disposing) { }
+        [System.Obsolete("Rfc2898DeriveBytes replaces PasswordDeriveBytes for deriving key material from a password and is preferred in new applications.")]
+#pragma warning disable 0809
         public override byte[] GetBytes(int cb) { throw null; }
```

Comment:
@bartonjs while admirable, I'm not sure we should be exposing this `Obsolete` on the ref for `PasswordDeriveBytes.GetBytes`. At the very least it should be a custom diagnostic that can be suppressed.

Reply:
This obsolete comes from .NET Framework:
https://referencesource.microsoft.com/#mscorlib/system/security/cryptography/passwordderivebytes.cs,139